src/damage.cpp

```diff
@@ -423,3 +423,30 @@ resistances load_resistances_instance( const JsonObject &jo )
     ret.resist_vals = load_damage_array( jo );
     return ret;
 }
+
+void damage_over_time_data::load_DoT_data( const JsonObject &obj )
```

Comment:
```suggestion
void damage_over_time_data::load( const JsonObject &obj )
```
should rename this and it should be good.